MediaBrowser.MediaEncoding/BdInfo/BdInfoDirectoryInfo.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using BDInfo.IO;
+using MediaBrowser.Model.IO;
+
+namespace MediaBrowser.MediaEncoding.BdInfo
+{
+    class BdInfoDirectoryInfo : BDInfo.IO.IDirectoryInfo
+    {
+        IFileSystem _fileSystem = null;
```

Comment:
what use has this `IFileSystem`? I was under impression we removed most of those abstractions in other places

Reply:
> Mostly looks good.
> 
> What package targets did you publish on Nuget? Got a link?

The package is published to https://www.nuget.org/packages/BDInfo/0.7.6.1 there's only the release BDInfo.dll for netstandard2.1 and netcoreapp3.0.

Reply:
> what use has this `IFileSystem`? I was under impression we removed most of those abstractions in other places

It seems to be still heavily used. This `_fileSystem` is needed for accesing the VFS which Jellyfin in exposing. (e.g. listing files/dirs, opening the filestreams).